api/app/signals/apps/signals/migrations/0142_auto_20210727_1406.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.5 on 2021-07-27 12:06
```

Comment:
Please fix this migration to match the model. We want to keep the amount of migrations to a minimum per PR.

Reply:
@vanbuiten Seeing the comments on migration 0142 can i assume this will be done in a sperate PR and i can ignore this one?